packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -239,6 +233,137 @@ class OpenSettingsAction extends StatelessWidget {
   }
 }
 
+class DevToolsAboutDialog extends StatefulWidget {
```

Comment:
should we name this `DevToolsInfoDialog` for consistency with `DevToolsInfoAction`?